R/zzz.R

```diff
@@ -10,19 +10,23 @@ R7_object <- new_class(
   }
 )
 
+new_base_class <- function(name) {
+  R7_class(name = name, constructor = function(.data) new_object(.data))
```

Comment:
I think the constructor argument has to be called `.data` (since that's what the property is) and they shouldn't have a default for consistency with the other R7 objects.